packages/build-utils/src/lambda.ts

```diff
@@ -12,7 +12,7 @@ interface Environment {
   [key: string]: string;
 }
 
-interface LambdaOptions {
+interface LambdaOptions<LauncherConfig = any> {
```

Comment:
Do we need `= any` here? I think passing in a generic should be sufficient. Or are you thinking it might be a breaking change 🤔

Reply:
Switched to using a subclass instead of a generic.